[32mINFO    [0m pytest_logger:log_util.py:49 File 'C:\Users\SandroJimena\PycharmProjects\Contact_List_App_Selenium_Python\test_data\test_data.json' exists, proceeding to read data.
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Retrieved value: {'email': 'testing_userrr@fakeee.com', 'password': '1234567', 'result': 'Contact List'}
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('css selector', '#email') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Input text value: 'testing_userrr@fakeee.com into element with locator: ('css selector', '#email')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'password') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Input text value: '1234567 into element with locator: ('id', 'password')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'submit') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Clicking the element ('id', 'submit')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('xpath', "//h1[text()='Contact List']") is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('xpath', "//h1[text()='Contact List']") is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Actual value of attribute is: Contact List
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Actual value is: Contact List while Expected value is: Contact List
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'add-contact') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Clicking the element ('id', 'add-contact')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'firstName') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Input text value: ' into element with locator: ('id', 'firstName')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'lastName') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Input text value: 'Jones into element with locator: ('id', 'lastName')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'birthdate') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Input text value: '1968-09-01 into element with locator: ('id', 'birthdate')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'email') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Input text value: 'jeffersonderrick@example.net into element with locator: ('id', 'email')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'phone') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Input text value: '2314258424 into element with locator: ('id', 'phone')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'street1') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Input text value: '472 Schmidt Inlet into element with locator: ('id', 'street1')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'street2') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Input text value: 'Apt. 198 into element with locator: ('id', 'street2')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'city') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Input text value: 'Davisbury into element with locator: ('id', 'city')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'stateProvince') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Input text value: 'Colorado into element with locator: ('id', 'stateProvince')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'postalCode') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Input text value: '07693 into element with locator: ('id', 'postalCode')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'country') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Input text value: 'Maldives into element with locator: ('id', 'country')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'submit') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Clicking the element ('id', 'submit')
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'error') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Element with locator: ('id', 'error') is visible within the timeout period
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Actual value of attribute is: Contact validation failed: firstName: Path `firstName` is required.
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully
[32mINFO    [0m pytest_logger:log_util.py:49 Actual value is: Contact validation failed: firstName: Path `firstName` is required. while Expected value is: Contact validation failed: firstName: Path `firstName` is required.
[32mINFO    [0m pytest_logger:log_util.py:49 Screenshot saved successfully